content/user/vulns/ManageVulns.md

```diff
@@ -26,6 +26,32 @@ Contrast provides multiple ways to narrow findings and focus on the vulnerabilit
 
 <a href="assets/images/Vulnerability_Filters.png" rel="lightbox" title="Vulnerability Filters"><img class="thumbnail" src="assets/images/Vulnerability_Filters.png"/></a>
 
+## Pending Vulnerabilities 
+
+If an administrator [requires administrative approval](admin-orgsettings.html#app-defaults) to close vulnerabilities, vulnerabilities with their specified statuses and severities go into a **Pending** state when you attempt to close them. Administrative approval applies to changes to vulnerabilities by two-way [bugtracker integrations](admin-orgintegrations.html#bugtracker) as well as [auto-remediation policies](admin-policymgmt.html#remediate).  
+
+When you attempt to close a vulnerability that an administrator must approve, you must provide a reason and justification for the status change. Once completed, Contrast confirms that the closure is pending an administrator's review. 
+
+<a href="assets/images/Vulns-mark-as-dialog.png" rel="lightbox" title="Enter a reason and justification for a status change"><img class="thumbnail" src="assets/images/Vulns-mark-as-dialog.png"/></a>
```

Comment:
If you're showing Not A Problem here, I might still update the caption to say "explanation" so it's more generic.